launch_test.go

```diff
@@ -454,7 +454,7 @@ func TestCreateWorkspaceBadStat(t *testing.T) {
 
 	wantWorkspace := Workspace{}
 
-	workspace, err := createWorkspace(TestWorkspace, "screwdriver-cd", "launcher")
+	workspace, err := createWorkspace(false, TestWorkspace, "screwdriver-cd", "launcher")
```

Comment:
Don't we need the test case when  isLocal is true.

Reply:
Thx.
I added test for that case.